quickjs.c

```diff
@@ -18271,6 +18295,7 @@ static void free_token(JSParseState *s, JSToken *token)
 static void __attribute((unused)) dump_token(JSParseState *s,
                                              const JSToken *token)
 {
+    printf("%d:%d ", token->line_num, token->col_num);
```

Comment:
Debug leftover or intentional?

Reply:
Intentional! And it's a debug helper in the first place.